kotsadm/operator/pkg/client/client.go

```diff
@@ -316,12 +319,7 @@ func (c *Client) deployManifests(args ApplicationManifests) (*commandResult, err
 	c.imagePullSecret = args.ImagePullSecret
 	c.watchedNamespaces = args.AdditionalNamespaces
 
-	result, err := c.ensureResourcesPresent(args)
```

Comment:
i like the old format since it has an additional error wrap and could be helpful when debugging